File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int cnt = 0, tm = 0;
        for(int i = 0; i < nums.length; i++){
            if(nums[i] == 1){
                tm++;
                cnt = Math.max(tm,cnt);
            } 
            else{
               
                tm = 0;
            }
        }
        return cnt;
    }
}

File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let cnt = 0, tm = 0;
        for (let i = 0; i < nums.length; i++) {
            if (nums[i] == 1) {
                tm++;

            } else {
                cnt = Math.max(tm, cnt);
                tm = 0;
            }
        }
        if (tm > cnt)
            cnt = tm;
        return cnt;
};